cppapi/server/multiattribute.h

```diff
@@ -47,7 +47,7 @@ class DeviceClass;
 
 struct EventPar
```

Comment:
this struct name is unclear to me, is it typo? Should it be `EventPair`? Or is it something completely different? Anyway it says nothing from its name concerning its purpose etc

Reply:
Thanks Igor!. Some time ago I started to refactor this area but dropped the PR as it grew too big. Here is the relevant commit that re-names `EventPar` to `AttributeEventSubscriptionState`.
https://github.com/tango-controls/cppTango/pull/572/commits/6b5194319575a922d1360b9a977cecb6116f65c3

Do you want me to do the rename in this PR?

Reply:
`AttributeEventSubscriptionState` sounds much more better! Yes, please rename. Thanks!

Reply:
I'v re-based the PR on tango-9-lts and re-named `EventPar` to `EventSubscriptionState` (without `Attribute` prefix as there is still a flag for device interface chagne event. Splitting this class requires further refactoring).